.github/workflows/bench.yml

```diff
@@ -0,0 +1,124 @@
+name: benches
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - dev
+  pull_request:
```

Comment:
should this run on pull_request or just dev/workflow_dispatch like wry?

Reply:
yeah let's move it to push dev only I think it's the best